app/assets/javascripts/discourse/app/components/light-dark-img.gjs

```diff
@@ -0,0 +1,54 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+import CdnImg from "discourse/components/cdn-img";
+import { getURLWithCDN } from "discourse-common/lib/get-url";
+
+export default class LightDarkImg extends Component {
```

Comment:
I created this component because this pair of light/dark images can be used in the future in other places.

I'm just not sure about the name, I couldn't think of anything better.

Reply:
I can't think of a better name, either.